docs/terminal/williamtown.md

```diff
@@ -70,6 +70,22 @@ Voiceless for all aircraft:
 
 All other aircraft coming from ARL CTA will be **Heads-up** Coordinated to WLM TCU.
 
+### Stereo Approaches (Fast Jet Aircraft)
+Fast Jet Aircraft typically arrive back into Willy from Gate 6, at 'F125', as such you can assign them the stereo approach to arrive back into Willy.
+
+!!! Example
+**MAPL45**: "WLM TCU, RTBing via G6 for RWY12"
+**WLM TCU**: "MAPL45, cleared the stereo appraoch for RWY 12 via G6, once 9DME north of the tacan descend not below `A050`"
+**MAPL45**: "Once 9DME north, descend not below `A050`"
+
+<figure markdown>
```

Comment:
Could we please remove this diagram, we're just avoiding including any copyright images/documents. Links to them are fine. Cheers

Reply:
Will do